src/dds/rs-dds-device-proxy.h

```diff
@@ -51,7 +52,12 @@ class dds_device_proxy : public software_device
 public:
     dds_device_proxy( std::shared_ptr< context > ctx, std::shared_ptr< realdds::dds_device > const & dev );
 
+    void tag_default_profiles( std::shared_ptr< stream_profile_interface > & profile,
```

Comment:
There's the other function, `tag_profiles`, that makes this confusing in that it's not differentiated.
I suggest to make it singular. Maybe something like `add_tags_to_profile`?

Reply:
Renamed `tag_default_profile_of_stream`.
Needed to override `tag_profiles` because default behavior of `device` removes the DDS set tags.